docs/HTTP3.md

```diff
@@ -233,6 +233,14 @@ Build curl:
      % make
      % make install
 
+You can build curl with cmake:
```

Comment:
Let's add an empty line here to match existing entries and make this render a code block:
```suggestion
You can build curl with cmake:

```

https://github.com/curl/curl/blob/master/docs/HTTP3.md

Reply:
Done.